pkg/apis/discovery/validation/validation_test.go

```diff
@@ -537,9 +594,10 @@ func TestValidateEndpointSliceUpdate(t *testing.T) {
 	standardMeta := metav1.ObjectMeta{Name: "es1", Namespace: "test"}
 
 	testCases := map[string]struct {
-		expectedErrors   int
-		newEndpointSlice *discovery.EndpointSlice
-		oldEndpointSlice *discovery.EndpointSlice
+		expectedErrors      int
+		nodeNameGateEnabled bool
+		newEndpointSlice    *discovery.EndpointSlice
```

Comment:
really minor nit, can we put oldEndpointSlice above newEndpointSlice, just makes the test easier to read

Reply:
yeah that was really bugging me too, but I wasn't sure I wanted to make the diff that big. I think it is a big enough improvement that I'll add it as a separate commit.

Reply:
Updated in latest commit.